waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -861,8 +876,32 @@ proc mountRlnRelay*(node: WakuNode, conf: WakuNodeConf) {.raises: [Defect, Value
       let memKeyPair = keyPair.toMembershipKeyPairs()[0]
       # mount the rln relay protocol in the on-chain/dynamic mode
       waitFor node.mountRlnRelayDynamic(memContractAddr = ethMemContractAddress, ethClientAddr = ethClientAddr, memKeyPair = some(memKeyPair), memIndex = some(rlnRelayIndex), ethAccAddr = ethAccountAddr, ethAccountPrivKey = ethAccountPrivKey, pubsubTopic = conf.rlnRelayPubsubTopic, contentTopic = conf.rlnRelayContentTopic)
+    elif fileExists("keyPair.txt") and fileExists("rlnIndex.txt"):
+      info "keyPair and rlnIndex files exist"
+      #With regards to printing the keys, it is purely for debugging purposes so that the user becomes explicitly aware of the current keys in use when nwaku is started.
+      #Note that this is only until the RLN contract being used is the one deployed on Goerli testnet.
+      #These prints need to omitted once RLN contract is deployed on Ethereum mainnet and using valuable funds for staking.
+      
+      let entireKeyPairFile = readFile("keyPair.txt")
+
+      let jsonObjectKeyPair = parseJson(entireKeyPairFile)
+      let deserializedKeyPair = to(jsonObjectKeyPair, MembershipKeyPair)
+
+      info "Deserialized key pair", keyPair=deserializedKeyPair
+      echo deserializedKeyPair 
+
+      
+      let entireRlnIndexFile = readFile("rlnIndex.txt")
+
+      let jsonObjectRlnIndex = parseJson(entireRlnIndexFile)
+      let deserializedRlnIndex = to(jsonObjectRlnIndex, MembershipIndex)
+
+      info "Deserialized rln index", rlnIndex=deserializedRlnIndex
+      echo deserializedRlnIndex 
+      waitFor node.mountRlnRelayDynamic(memContractAddr = ethMemContractAddress, ethClientAddr = ethClientAddr, memKeyPair = some(deserializedKeyPair), memIndex = some(deserializedRlnIndex), ethAccAddr = ethAccountAddr, ethAccountPrivKey = ethAccountPrivKey, pubsubTopic = conf.rlnRelayPubsubTopic, contentTopic = conf.rlnRelayContentTopic)
     else:
       # no rln credential is provided
       # mount the rln relay protocol in the on-chain/dynamic mode
+      info("no rln credential is provided") 
```

Comment:
If possible, try to stick to the same syntax convention as above for logging, just to improve maintainability and consistency:
```suggestion
      info "no rln credential is provided"
```